src/pages/animations.html

```diff
@@ -4,7 +4,7 @@
 
 
 <section class="site-section">
-  <h3 class="site-subheadline">Animation Types</h3>
+  <h3 class="site-subheadline">Animation Tips</h3>
```

Comment:
Did you mean to change this to Tips?

Reply:
Haha, was testing to make sure changes were coming through. Ill fix that right quick